content/en/docs/concepts/configuration/manage-resources-containers.md

```diff
@@ -463,7 +463,7 @@ that file but the kubelet does not categorize the space as in use.
 {{% /tab %}}
 {{% tab name="Filesystem project quota" %}}
 
-{{< feature-state for_k8s_version="v1.15" state="alpha" >}}
+{{< feature-state for_k8s_version="v1.31" state="beta" >}}
```

Comment:
Consider using the "feature_gate_name" option for `feature-state` shortcode. This will make it more data-driven by pulling from the feature gate description file.

```suggestion
{{< feature-state feature_gate_name="LocalStorageCapacityIsolationFSQuotaMonitoring" >}}
```